flocker/control/script.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script for starting control service server.
+"""
+
+from twisted.python.usage import Options
+from twisted.internet.endpoints import TCP4ServerEndpoint
+
+from .httpapi import create_api_service
+from ..common.script import (
+    flocker_standard_options, FlockerScriptRunner, main_for_service)
+
+
+@flocker_standard_options
+class ControlOptions(Options):
+    """
+    Command line options for ``flocker-control`` cluster management process.
+    """
+    optParameters = [
+        # XXX also choose where configuration will be stored
```

Comment:
This will be a config file like the current `/etc/flocker/config.yml` right?  And we're going to gradually expand that to cover the rest of the service (not deployment or application) configuration data we need to track?

Eventually perhaps the config file should be the only thing accepted on the command line and everything else, like port numbers and certificate/key paths, should be in the config file.  But that's definitely eventually, not right away.  Which raises the question - does this branch need the config file at all?  If the control service accepts a port number to listen on as a command line argument, what other configuration does it need now?  I can't think of anything.  (So maybe punt config file handling down the line a bit.)